docs/modules/reference/pages/service-assurance/monitors/SSLCertMonitor.adoc

```diff
@@ -95,38 +95,50 @@ Configuration in `poller-configuration.xml` is as follows:
 [source, xml]
 ----
 <service name="SSL-Cert-IMAPS-993" interval="7200000" user-defined="false" status="on">
-    <parameter key="retry" value="2"/>
-    <parameter key="timeout" value="2000"/>
-    <parameter key="port" value="993"/>
-    <parameter key="days" value="30"/>
+    <parameter key="retry" value="2"/> <1>
+    <parameter key="timeout" value="2000"/> <2>
+    <parameter key="port" value="993"/> <3>
+    <parameter key="days" value="30"/> <4>
 </service>
+
 <service name="SSL-Cert-SMTPS-465" interval="7200000" user-defined="false" status="on">
-    <parameter key="retry" value="2"/>
-    <parameter key="timeout" value="2000"/>
-    <parameter key="port" value="465"/>
-    <parameter key="days" value="30"/>
+    <parameter key="retry" value="2"/> <1>
+    <parameter key="timeout" value="2000"/> <2>
+    <parameter key="port" value="465"/> <3>
+    <parameter key="days" value="30"/> <4>
 </service>
+
 <service name="SSL-Cert-HTTPS-443" interval="7200000" user-defined="false" status="on">
-    <parameter key="retry" value="2"/>
-    <parameter key="timeout" value="3000"/>
-    <parameter key="port" value="443"/>
-    <parameter key="days" value="30"/>
-    <parameter key="server-name" value="${nodelabel}.example.com"/>
+    <parameter key="retry" value="2"/> <1>
+    <parameter key="timeout" value="3000"/> <2>
+    <parameter key="port" value="443"/> <3>
+    <parameter key="days" value="30"/> <4>
+    <parameter key="server-name" value="${nodelabel}.example.com"/> <5>
 </service>
+
 <service name="XMPP-STARTTLS-5222" interval="7200000" user-defined="false" status="on">
-    <parameter key="retry" value="2"/>
-    <parameter key="timeout" value="3000"/>
-    <parameter key="port" value="5222"/>
-    <parameter key="days" value="30"/>
-    <parameter key="starttls-preamble" value="<stream:stream xmlns:stream='http://etherx.jabber.org/streams' xmlns='jabber:client' to='{ipAddr}' version='1.0'>"/>
-    <parameter key="starttls-preamble-response" value="^.*starttls.*$"/>
-    <parameter key="starttls-start" value="<starttls xmlns='urn:ietf:params:xml:ns:xmpp-tls'/>"/>
-    <parameter key="starttls-start-response" value="^.*starttls.*$"/>
+    <parameter key="retry" value="2"/> <1>
+    <parameter key="timeout" value="3000"/> <2>
+    <parameter key="port" value="5222"/> <3>
+    <parameter key="days" value="30"/> <4>
+    <parameter key="starttls-preamble" value="<stream:stream xmlns:stream='http://etherx.jabber.org/streams' xmlns='jabber:client' to='{ipAddr}' version='1.0'>"/> <6>
```

Comment:
This is one where the wrap doesn't work, so we don't see the number, but ok for now. Fixing it will be more effort and should be in a separate PR, as it is a universal config in the docs css.